src/explorer/model/Dependencies.ts

```diff
@@ -0,0 +1,77 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+import * as vscode from "vscode";
+import { ITreeItem } from "./ITreeItem";
+
+const DUPLICATE_VALUE: string = "omitted for duplicate";
+const CONFLICT_VALUE: string = "omitted for conflict";
+
+export class Dependencies implements ITreeItem {
+    public dependency: string;
+    public pomPath: string;
+    private curDep: string;
+    private separator: string;
+    private collapsedState: vscode.TreeItemCollapsibleState;
+
+    constructor(dependency: string, separator: string, collapsedState: vscode.TreeItemCollapsibleState, pomPath: string) {
+        this.dependency = dependency;
+        this.separator = `${separator}   `; // three spaces
+        this.collapsedState = collapsedState;
+        this.pomPath = pomPath;
+    }
+
+    public getContextValue(): string {
+        return "Dependencies";
+    }
+
+    public async getChildren(): Promise<Dependencies[] | undefined> {
+        return Promise.resolve(this.getDepsInString(this.dependency));
+    }
+
+    public getTreeItem(): vscode.TreeItem | Thenable<vscode.TreeItem> {
+        this.curDep = this.dependency.split(this.separator, 1)[0];
```

Comment:
1. parsing work should be extracted into another utility.
2. during parsing with some complicated regular expression, it's better to add comments about i) a sample input and ii) the matched parts.